keepalived/vrrp/vrrp_snmp.c

```diff
@@ -1821,13 +1820,18 @@ vrrp_snmp_instance(struct variable *vp, oid *name, size_t *length,
 		return (u_char *)&long_ret;
 
 	case VRRP_SNMP_INSTANCE_USELVSSYNCDAEMON:
+                long_ret.u = 0;
+#ifdef _WITH_LVS_
 		long_ret.u = (global_data->lvs_syncd.vrrp == rt)?1:2;
+#endif
 		return (u_char *)&long_ret;
 	case VRRP_SNMP_INSTANCE_LVSSYNCINTERFACE:
+#ifdef _WITH_LVS_
```

Comment:
See earlier comments re how conditional compilation is normally handled in keepalived for SNMP objects not defined.